### PR TITLE
Added testing functionality

### PR DESCRIPTION
**Pull request by nickciolpan**
ClickUp link: https://app.clickup.com/t/3cnntc

- [x] Passes all regression tests
- [x] Is cross-browser compatible
- [x] Is cross-device compatible
- [x] Fullfils General Acceptance Criteria (see Wiki)
- [x] Fullfils Specific Acceptance Criteria (see ClickUp car)
- [x] Adheres to code standards
- [x] All code review has been addressed
- [x] No linting errors

TODO:

- ~~[ ] Test against Design files functionality~~
- [x] Run Selenium in a background job
- ~~[ ] Add Safari headless browser~~
- [x] Add our most supported breakpoints
- [x] Write documentation

To understand how to fill in the above, see: https://wiki.graffino.net/doc/quality-assurance-KPkwZGTCJj

### CODE REVIEW

File: .hermione.conf.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
+module.exports = {
+  sets: {
+    desktop: {
+      files: 'tests/desktop'
+    }
+  },
+  baseUrl: 'http://localhost:3000',
+  gridUrl: 'http://localhost:4444/wd/hub',
+  compositeImage: true,
+  screenshotsDir: 'tests/hermione/screens',
```
Comment (zenopopovici) [inline .hermione.conf.js]: Clearer name for reports?

File: .hermione.conf.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        chromeOptions: {
+          args: ['--headless'],
+        },
+      },
+      windowSize: {
+        width: 414,
+        height: 800
+      },
+    }
+    // safariXL: {
```
Comment (zenopopovici) [inline .hermione.conf.js]: Remove comments

File: package.json [hunk 3]
```diff
@@ -15,7 +15,9 @@
     "wp-update": "composer update && node bin/sync-uploads.js --no-confirm",
     "wp-sync-uploads": "node bin/sync-uploads.js",
     "wp-clean-install-folder": "node bin/clean-wp.js",
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test-prep": "./node_modules/.bin/selenium-standalone install && ./node_modules/.bin/selenium-standalone start",
```
Comment (zenopopovici) [inline package.json]: `test-setup` would be a clearer name as we have `wp-setup` ...

File: package.json [hunk 4]
```diff
@@ -15,7 +15,9 @@
...
+    "test": "node node_modules/hermione/bin/hermione",
+    "approve": "node node_modules/hermione/bin/hermione --update-refs",
```
Comment (zenopopovici) [inline package.json]: should be `test-approve`

**Review by zenopopovici — CHANGES_REQUESTED** [review]

**Comment by nickciolpan** [comment]
@zenopopovici Merge this please if nothing else.

**Review by pauljugarean — APPROVED** [review]

**Review by zenopopovici — APPROVED** [review]

**Pull request merged**